_posts/blog/2016-11-16-simpler-scripts-and-config.md

```diff
@@ -0,0 +1,163 @@
+```
+
+Below are some notes on this change:
+
+* The 'accumulo' script was left alone except for improved usage and the addition of 'create-config' and 'build-native'
+  commands to replace 'bootstrap_config.sh' and 'build_native_library.sh'.
+* The 'accumulo-service' script was created to manage Accumulo processes as services
+* The 'accumulo-cluster' command was created to manage Accumulo on cluster and replaces 'start-all.sh' and 'stop-all.sh'.
+* All optional scripts in `bin/` were moved to `contrib/`:
+
+      $ ls accumulo-2.0.0/contrib/
+      bootstrap-hdfs.sh  check-tservers  gen-monitor-cert.sh  tool.sh
+
+### Less configuration
+
+Before 2.0.0, Accumulo's `conf/` directory looked like the following (after creating initial config files
+using 'bootstrap_config.sh'):
+
+```bash
+$ ls accumulo-1.8.0/conf/
+accumulo-env.sh          auditLog.xml  generic_logger.properties            masters                    slaves
+accumulo-metrics.xml     client.conf   generic_logger.xml                   monitor                    templates
+accumulo.policy.example  examples      hadoop-metrics2-accumulo.properties  monitor_logger.properties  tracers
+accumulo-site.xml        gc            log4j.properties                     monitor_logger.xml
+```
+
+While all of these files have a purpose, many are only used in rare situations. Therefore, the
+'accumulo create-config' (which replaces 'bootstrap_config.sh') now only generates a minimum
+set of configuration files needed to run Accumulo.
+
+```bash
+$ cd accumulo-2.0.0
+$ ./bin/accumulo create-config
+$ ls conf/
+accumulo-env.sh  accumulo-site.xml  client.conf  examples
+```
+
+The 'accumulo create-config' command does not generate host files (i.e 'tservers', 'monitor', etc) to run processes locally.
+These files are only required by the 'accumulo-cluster' command which has a command to generate them.
+
+```bash
+$ cd accumulo-2.0.0/
+$ ./bin/accumulo-cluster create-config
+$ ls conf/
+accumulo-env.sh  accumulo-site.xml  client.conf  examples  gc  masters  monitor  tracers  tservers
+```
+
+Any less common configuration files that were not generated above can still be found in `conf/examples`.
+
+### Better usage
+
+Before 2.0.0, the 'accumulo' command had a limited usage:
+
+```
+$ ./accumulo-1.8.0/bin/accumulo
+accumulo admin | check-server-config | classpath | create-token | gc | help | info | init | jar <jar> [<main class>] args |
+  login-info | master | minicluster | monitor | proxy | rfile-info | shell | tracer | tserver | version | zookeeper | <accumulo class> args
+```
+
+For 2.0.0, all 'accumulo' commands were given a short description and organized into the groups.
+
+```
+$ ./accumulo-2.0.0/bin/accumulo help
+
+Usage: accumulo <command> (<argument> ...)
+
+Core Commands:
+  create-config                  Create Accumulo configuration
+  build-native                   Build Accumulo native libraries
+  init                           Initializes Accumulo
```

Comment:
I'm just noticing the inconsistency in the description:

> Create Accumulo configuration

> Initializes Accumulo

I assume this would be a tweaking of the scripts themselves and then later updating this post?

Personally, because these are all actions, they should all be 3rd-person:

> Creates Accumulo configuration
> Builds Accumulo native libraries
> Prints Accumulo version